src/c2goto/library/time.c

```diff
@@ -6,10 +6,12 @@
 #include <time.h>
 #undef time
 
+time_t __VERIFIER_nondet_time_t();
```

Comment:
Could you please add the declaration of this function in:

- https://github.com/esbmc/esbmc/blob/master/src/clang-c-frontend/clang_c_language.cpp#L319
- https://github.com/esbmc/esbmc/blob/master/src/ansi-c/ansi_c_language.cpp#L75